spec/models/wunderground_history_spec.rb

```diff
@@ -37,10 +37,21 @@
   end
 
   describe "populate_observations!" do
+    it "stores an ObservationCollection object to observations attribute" do
+      wh.time = Time.zone.parse('January 1, 2000 00:00:00 -04:00')
+      wh.response = wunderground.history_for(wh.time, 'knyc')
+      expect(wh.observations).to be_nil
+      wh.populate_observations!
+      expect(wh.observations).to be_a ObservationCollection
```

Comment:
again better to test behavior or what that collection is expected to have instead